project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
 
   val SjsMacroTaskExecutor = Def.setting("org.scala-js" %%% "scala-js-macrotask-executor" % "1.0.0")
   val SjsSecureRandom = Def.setting("org.scala-js" %%% "scalajs-java-securerandom" % "1.0.0")
-  val logback = "ch.qos.logback" % "logback-classic" % "1.3.5"
+  val logback = "ch.qos.logback" % "logback-classic" % "1.2.11"
```

Comment:
Why did we downgrade on the `logback-classic` version?

Reply:
Because I didn't want to force users into accepting an slf4j version change